oauth2-server/src/main/java/com/clouway/oauth2/DateTime.java

```diff
@@ -1,11 +1,12 @@
 package com.clouway.oauth2;
 
+import java.io.Serializable;
 import java.util.Date;
 
 /**
  * @author Miroslav Genov (miroslav.genov@clouway.com)
  */
-public final class DateTime {
+public final class DateTime implements Serializable {
```

Comment:
As it acts as a primitive (wrapper of date) someone would be able to use it for serialization (memcached, datastore and etc).